packages/fetch/index.js

```diff
@@ -1,19 +1,26 @@
 'use strict';
 
-const parseBody = (response) => {
+const parseBody = async (response) => {
+    let body = '';
+
+    for await (const chunk of response.body) {
+        body += new TextDecoder().decode(chunk);
+    }
```

Comment:
This might be slightly more performant?

```suggestion
    const decoder = new TextDecoder();
    let body = '';

    for await (const chunk of response.body) {
        body += decoder.decode(chunk);
    }
```